src/pages/AccountSettings/tabs/OktaAccess/OktaConfigForm/OktaConfigForm.test.tsx

```diff
@@ -142,9 +139,7 @@ describe('OktaConfigForm', () => {
     const clientSecretInput = await screen.findByLabelText(/Client Secret/)
     await userEvent.type(clientSecretInput, 'clientSecret')
 
-    const eyeIcon = await screen.findByRole('button', {
-      name: /eye/,
-    })
+    const eyeIcon = await screen.findByTestId('toggle-password')
```

Comment:
nit: is this variable name and at line 155 still accurate/helpful now that we're not finding by "eye"?

Reply:
Ahh, just reached the next file. The eye is still in there. Oh well, feel free to just leave as is